#tryexcept
print("enter num 1")
num1 = input()
print(" enter num 2 ")
num2 = input()11
try:
    print ("the sum of these numbers is ",
           int(num1) + int(num2))
except exception as e:
    print(e)



print("this line is very imp brother!")